service/matching/physical_task_queue_key.go

```diff
@@ -128,12 +144,16 @@ func (q *PhysicalTaskQueueKey) PersistenceName() string {
 	case *tqid.NormalPartition:
 		baseName := q.TaskQueueFamily().Name()
 
-		if len(q.versionSet) > 0 {
-			return nonRootPartitionPrefix + baseName + partitionDelimiter + q.versionSet + versionSetDelimiter + strconv.Itoa(p.PartitionId())
+		if len(q.version.versionSet) > 0 {
+			return nonRootPartitionPrefix + baseName + partitionDelimiter + q.version.versionSet + versionSetDelimiter + strconv.Itoa(p.PartitionId())
 		}
 
-		if len(q.buildId) > 0 {
-			encodedBuildId := base64.URLEncoding.EncodeToString([]byte(q.buildId))
+		if len(q.version.deploymentSeriesName) > 0 {
+			encodedBuildId := base64.URLEncoding.EncodeToString([]byte(q.version.buildId))
```

Comment:
Can we use RawURLEncoding to avoid the useless padding chars? I guess we can't change the v2 names but we can change these